source/includes/markdown/_13_1-scim.html.md

```diff
@@ -31,10 +58,23 @@ The accepted user attributes are:
 |name.givenName|string|Unsupported, use `formatted`|
 |name.familyName|string|Unsupported, use `formatted`|
 |name.formatted|string|The full name of the user.|
-|emails|complex|Email addresses for the user.|
+|emails|multi-valued complex|Email addresses for the user.|
```

Comment:
added "multi-valued" to distinguish it from "name" which is complex but NOT multi-valued.